mm2src/coins/eth/eth_swap_v2.rs

```diff
@@ -87,7 +114,7 @@ impl EthCoin {
                         })?;
                 }
                 self.sign_and_send_transaction(
-                    U256::from(0),
+                    0.into(),
```

Comment:
The eth tpu v2 code in the previous PR branch was in a very early stage. While the previous PR was under review, I continued working on the swap v2 implementation in the current branch to avoid making the previous PR larger. In the new functions, I used `0.into()` and noticed that the old function had `U256::from(0)`. decided to change it to `0.into()` as well to maintain a consistent style.

Reply:
Nothing serious - Doing things in explicit way is always a better option for me.

Reply:
When you dont trust compiler :)
used U256::from https://github.com/KomodoPlatform/komodo-defi-framework/pull/2169/commits/f6ffcf1bc90a5ad2e5da720a5435331047390806

Reply:
It's actually for making the reviews/debugging easier :smile: